packages/react-dom/src/__tests__/ReactIdentity-test.js

```diff
@@ -236,14 +252,24 @@ describe('ReactIdentity', () => {
     const instance0 = <span key="A" />;
     const instance1 = <span key="B" />;
 
-    let wrapped = <TestContainer first={instance0} second={instance1} />;
-
-    wrapped = ReactDOM.render(wrapped, document.createElement('div'));
+    const container = document.createElement('div');
+    const root = ReactDOMClient.createRoot(container);
+    let wrappedRef = React.createRef();
+    await act(async () => {
+      root.render(
+        <TestContainer first={instance0} second={instance1} ref={wrappedRef} />,
+      );
+    });
+    const wrapped = wrappedRef.current;
+    // There isn't an alternative to findDOMNode for this:
```

Comment:
Is this actually true? Is there another way around using findDOMNode here??

Reply:
Not unless you know the render() implementation. But since we know that in this test, `container.firstChild` should do. That's what we've been using in other tests as well.

Reply:
Thanks for the tip! Fixed.